lib/internal/Magento/Framework/Escaper.php

```diff
@@ -262,27 +262,6 @@ public function escapeCss($string)
         return $this->getEscaper()->escapeCss($string);
     }
 
-    /**
-     * Escape quotes in java script
-     *
-     * @param string|array $data
-     * @param string $quote
-     * @return string|array
-     * @deprecated 100.2.0
-     */
-    public function escapeJsQuote($data, $quote = '\'')
```

Comment:
I am going to check how often, if at all, these methods are used in the Marketplace extensions before progressing here.